tt_metal/host_api.hpp

```diff
@@ -26,6 +26,17 @@
 class CoreRange;
 class CoreRangeSet;
 
+// Adding to tt::tt_metal namespace as we transition to moving this out of global namespace eventually.
+namespace tt::tt_metal {
+   using ::CoreRange;
+   using ::CoreRangeSet;
+}
```

Comment:
I think this should go inside `core_coord.h` together with what you did for `CoreCoord` instead of putting it here.

Reply:
Done. `CoreRange, CoreRangeSet, CoreCoord`  are all placed in  `core_coord.h`.